manufacturing/enclosed_build/README.md

```diff
@@ -1,21 +1,48 @@
 # Enclosed build
 
```

Comment:
Put a disclaimer here about not using on patients yet like is on the Pizza Build page.